background.js

```diff
@@ -1,3 +1,19 @@
 chrome.runtime.onInstalled.addListener(() => {
-
+    
 });
+
+//lists items that should be added to list when popup opens
+let itemsToAdd = [];
```

Comment:
You can actually use `const` when you're declaring an array, even if you intend to edit its contents. You're not actually _mutating_ the object (array) reference, rather, you're editing the _contents_ of the array.

There's some more info on that here: https://discuss.codecademy.com/t/faq-arrays-arrays-with-let-and-const/371595/3